server/api/api.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const Sighting = require('../models/Sighting.js');
+const sightingsController = require('../lib/sightingsController.js');
+
+const router = express.Router();
+
+// TODO: Adjust API to return data or err instead of unnamed JSON
```

Comment:
Is this TODO still valid? We want to avoid changing return signatures whenever possible

Reply:
This is still active. I added it based on your recipe book, so if it's best practice to have all returns named the same thing, then we can add that change. If it doesn't matter and more of a styling choice that you made years ago, then we can drop it.

Reply:
👍

Reply:
Updated response JSON and adjusted components to receive named JSON. Removed TODO